plugins/ilg.gnumcueclipse.managedbuild.cross.arm/build.properties

```diff
@@ -28,3 +29,4 @@ bin.includes = META-INF/,\
                about.ini,\
                about.mappings,\
                about.properties
+src.includes = build.properties
```

Comment:
Is this correct? In the RISC-V file you added `about.html` and `schema`.

Reply:
Thanks for catching this, will fix

Reply:
Done